airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java

```diff
@@ -84,10 +84,11 @@ public String uploadRecordsToStage(final JdbcDatabase database,
       throw new RuntimeException(
           String.format("Exceptions thrown while uploading records into stage: %s", Strings.join(exceptionsThrown, "\n")));
     }
+    LOGGER.info("Successfully loaded records to stage {} with {} tries", stagingPath, exceptionsThrown.size());
     return recordsData.getFilename();
   }
 
-  private void loadDataIntoStage(final JdbcDatabase database, final String stageName, final String stagingPath, final SerializableBuffer recordsData)
+  private void uploadRecordsToBucket(final JdbcDatabase database, final String stageName, final String stagingPath, final SerializableBuffer recordsData)
```

Comment:
NOTE: this method is renamed because it follows a similarly named method in `S3StorageOperations` which takes in the same parameters, the previous name `loadDataIntoStage` _also_ exists within `S3StorageOperations` but uses a subset of these parameters which made it confusing

Renamed so they are "in sync" but not really cause this method doesn't return a String (fileName) like the other methods. The filename is retrieved _after_ successfully loading data into stage through the `recordsData.getFilename();` method on #L88 above